assets/source/sass/theme.scss

```diff
@@ -33,3 +33,7 @@
     @import '../../../custom-modules/ColoredIndex/assets/sass/colored-index';
     @import '../../../custom-modules/Teaser/assets/sass/teaser';
 }
+
+.is-subsite {
```

Comment:
Perhaps this should have both `.is-subsite.sinab` since the included file is for SINAB only? It would remove the need to have everything wrapped in `&.sinab` in the included CSS.

Reply:
More subsites will be added to this shortly, so I think the separaition is clearer this way, as we wont have to repeat the `is-subsite` class each time.